bitnami/spring-cloud-dataflow/templates/_helpers.tpl

```diff
@@ -276,7 +276,7 @@ Return the RabbitMQ Port
 */}}
 {{- define "scdf.rabbitmq.port" -}}
 {{- if .Values.rabbitmq.enabled }}
-    {{- printf "%d" (.Values.rabbitmq.service.port | int ) -}}
+    {{- printf "%d" (default 5672 .Values.rabbitmq.service.port | int ) -}}
```

Comment:
This is indeed a regression introduced in https://github.com/bitnami/charts/pull/9817, where the `.Values.rabbitmq.service.port` was replaced by `.Values.rabbitmq.service.ports.amqp`. The default value is `5672` already, so we might simply do as follows:

```suggestion
    {{- printf "%d" (.Values.rabbitmq.service.ports.amqp | int ) -}}
```